src/server/index.js

```diff
@@ -9,11 +9,124 @@
 const Koa = require('koa');
 const IO = require('koa-socket');
 const Redux = require('redux');
+import Crypto from 'crypto';
+import Router from 'koa-router';
+import koaBody from 'koa-body';
 import { InMemory, Mongo } from './db';
 import { createGameReducer } from '../core/reducer';
+
 const PING_TIMEOUT = 20 * 1e3;
 const PING_INTERVAL = 10 * 1e3;
 
+const isActionFromAuthenticPlayer = async ({
+  action,
+  db,
+  gameID,
+  playerID,
+}) => {
+  const gameMetadata = await db.get(`${gameID}:metadata`);
```

Comment:
I really don't like the duplication of creating the string metadata key in multiple places, so I need to pull that out.